JdeRobot/JdeRobot
# [issue #526] Added ROS compatibility library for sensors

To make the compilation of JdeRobot works, this PR has the need for the installation of some ROS libraries.

To test it:
- Install the following dependencies: `ros-jade-roscpp`, `ros-jade-std-msgs`, `ros-jade-message-generation`, `ros-jade-cv-bridge`, and `ros-jade-image-transport`. Maybe you also need `ros-jade-message-runtime`.
- As it appears in the ROS wiki, configure the ROS environment (more info [here):](http://wiki.ros.org/jade/Installation/Ubuntu#Initialize_rosdep)
   `sudo rosdep init`
   `rosdep update`
   `echo "source /opt/ros/jade/setup.bash" >> ~/.bashrc`
   `source ~/.bashrc`
- Compile as usual
- Launch the test following this video from @shady-cs15 https://www.youtube.com/watch?v=nLMnnO6Cl4k

Instead of camdriver and camclient you must use cameraserverROS and cameraviewROS.
